cloudinit/config/cc_keyboard.py

```diff
@@ -18,14 +18,14 @@
 
 DEFAULT_KEYBOARD_MODEL = "pc105"
 
-distros = distros.Distro.expand_osfamily(["arch", "debian", "redhat", "suse"])
+distros_ = distros.Distro.expand_osfamily(["arch", "debian", "redhat", "suse"])
```

Comment:
Nitpick: Why the underscore here? Is this to disambiguate from the import? The trailing underscore is a bit unusual, maybe we could use `supported_distros` or something like that instead?

Reply:
Exactly, the intention was to differentiate this variable name from the import. mypy doesn't like the same variable being a module and a list. It is also better like this if a future modification requires using the `distro` module.

Trailing underscore is used by convention to avoid conflicts with Python keyword: [PEP8](https://peps.python.org/pep-0008/#descriptive-naming-styles), which is not the case here.

Variable name improved. Thanks for this suggestion!